Tests/scripts/run_tests.sh

```diff
@@ -22,4 +22,8 @@ if [ $RETVAL -eq 0 ]; then
   touch "$filepath"
 fi
 
-exit $RETVAL
+if [ "$IS_NIGHTLY" = true ]; then
```

Comment:
in general it's a best practice to prevent shell string expansion by using `"${}"`
```suggestion
if [ "${IS_NIGHTLY}" = true ]; then
```
Also i don't understand why we don't fail this step in case it's the nightly?

Reply:
You asked (in the kickoff) not to fail the server step if it failed only on test playbooks on nightly, didn't you?

Reply:
@bziser ok, now i understand the flow, thanks. please only address the shell string issue.